deploy/helm/sumologic/templates/otelcol-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if eq .Values.sumologic.traces.enable true }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-otelcol" (include "sumologic.name" .) }}
+  labels:
+    app: {{ printf "%s-otelcol" (include "sumologic.labels.app" .) }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  minReadySeconds: 5
+  progressDeadlineSeconds: 120
+  replicas: {{ .Values.otelcolDeployment.replicas }}
+  selector:
+    matchLabels:
+      app: {{ printf "%s-otelcol" (include "sumologic.labels.app" .) }}
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "8888"
+        prometheus.io/scrape: "true"
```

Comment:
we use Prometheus Operator for metrics collection.  That is what we recommend customers deploy. When using the Operator, annotation-based scraping is not supported and you must use a ServiceMonitor instead.  here is how we get the Fluentd and fluent-bit metrics we insturment - https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/values.yaml#L336

Reply:
👍 done